src/components/templates/Playa/VenuePreview.tsx

```diff
@@ -10,6 +10,7 @@ import {
 } from "settings";
 import UserList from "components/molecules/UserList";
 import { usePartygoers } from "hooks/users";
+import { useSparkleFirestoreConnect } from "useSparkleFirestoreConnect";
```

Comment:
Is this import path correct? Shouldn't it be 

```suggestion
import { useSparkleFirestoreConnect } from "hooks/useSparkleFirestoreConnect";
```